Klarna.php

```diff
@@ -3651,13 +3651,24 @@ protected function xmlrpc_call($method, $array)
                 }
             }
 
-            //Send the message.
             $selectDateTime = microtime(true);
             if (self::$xmlrpcDebug) {
                 $this->xmlrpc->setDebug(2);
             }
+            //Save previous XML RPC client encoding settings.
+            $tmp_xmlrpc_defencoding = $GLOBALS['xmlrpc_defencoding'];
+            $tmp_xmlrpc_internalencoding = $GLOBALS['xmlrpc_internalencoding'];
+            //Set XML RPC client encoding settings.
+            $GLOBALS['xmlrpc_defencoding'] = 'ISO-8859-1';
+            $GLOBALS['xmlrpc_internalencoding'] = 'UTF-8';
```

Comment:
After some further investigation, the simplest solution (for us both), would be for you to just set ```php
$GLOBALS['xmlrpc_internalencoding'] = 'UTF-8';
```

After you have configured the Klarna object (that's when the xmlrpc.inc file is loaded by the composer autoload and the globals are set).

We do not want to change the default encoding due to backward compatibility reasons.